.gitignore

```diff
@@ -44,3 +44,8 @@ yarn-error.log*
 
 # Local Netlify folder
 .netlify
+
+# Cypress
+cypress/integration/1-getting-started
+cypress/integration/2-advanced-examples
```

Comment:
```suggestion
```

Please remove the folder completely instead of ignoring them, because those examples are not supposed to be there anyway. We should be using `e2e` folder for our integration tests.